File: src/main/java/com/atsuishio/superbwarfare/client/model/entity/Sc50Model.java
package com.atsuishio.superbwarfare.client.model.entity;

import com.atsuishio.superbwarfare.Mod;
import com.atsuishio.superbwarfare.entity.projectile.Sc50Entity;
import net.minecraft.resources.ResourceLocation;
import software.bernie.geckolib.model.GeoModel;

public class Sc50Model extends GeoModel<Sc50Entity> {

    @Override
    public ResourceLocation getAnimationResource(Sc50Entity entity) {
        return null;
    }

    @Override
    public ResourceLocation getModelResource(Sc50Entity entity) {
        return Mod.loc("geo/sc_50.geo.json");
    }

    @Override
    public ResourceLocation getTextureResource(Sc50Entity entity) {
        return Mod.loc("textures/entity/ju_87.png");
    }
}

File: src/main/kotlin/com/atsuishio/superbwarfare/client/model/entity/Sc50Model.kt
package com.atsuishio.superbwarfare.client.model.entity

import com.atsuishio.superbwarfare.Mod.Companion.loc
import com.atsuishio.superbwarfare.entity.projectile.Sc50Entity
import software.bernie.geckolib.model.GeoModel

class Sc50Model : GeoModel<Sc50Entity>() {
    override fun getAnimationResource(entity: Sc50Entity) = null

    override fun getModelResource(entity: Sc50Entity) = loc("geo/sc_50.geo.json")

    override fun getTextureResource(entity: Sc50Entity) = loc("textures/entity/ju_87.png")
}
